manim/mobject/graphing/functions.py

```diff
@@ -23,9 +24,9 @@ class ParametricFunction(VMobject, metaclass=ConvertToOpenGL):
     Parameters
     ----------
     function
-        The function to be plotted in the form of ``(lambda x: x**2)``
+        The function to be plotted in the form of ``(lambda t: (x(t), y(t), z(t)))``
     t_range
-        Determines the length that the function spans. By default ``[0, 1]``
+        Determines the length that the function spans in the form of [t_min, t_max, step=0.01]. By default ``[0, 1]``
```

Comment:
```suggestion
        Determines the length that the function spans in the form of [t_min, t_max[, step=0.01]]. By default ``[0, 1]``
```
To use the python notation for optional. What do you think? Does this add clarity, or would it be confusing for beginners?

Reply:
Hm...I think it would increase confusion; as someone who has pretty scarce Python experience, I might accidentally interpret is as perhaps a nested list or something.